infrastructure/async/src/main/java/tech/pegasys/teku/infrastructure/async/AsyncRunner.java

```diff
@@ -29,17 +29,27 @@
   <U> SafeFuture<U> runAfterDelay(
       ExceptionThrowingFutureSupplier<U> action, long delayAmount, TimeUnit delayUnit);
```

Comment:
(nit) m.b. remove all the `long delayAmount, TimeUnit delayUnit` variants in favor of `Duration`? Former seems to be kind of legacy style.

Reply:
Filed #3563

Reply:
Yeah, good idea.  Thanks for adding that ticket.